### PR TITLE
ObO presubmit test for OCP

### PR DESCRIPTION
**Pull request by lihongyan1**

### CODE REVIEW

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-ci-index-observability-bundle

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-ci-index-observability-bundle

File: ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.13"
+  steps:
+    test:
+    - as: install
+      cli: latest
+      commands: |
+        oc create namespace obo
+        operator-sdk run bundle -n obo "$OO_BUNDLE" --install-mode AllNamespaces --skip-tls
```
Comment (vprashar2929) [inline ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main.yaml:57]: `--install-mode` AllNamespaces doesn't seem to work fine on ProwCI. Simple `operator-sdk run bundle -n obo "$OO_BUNDLE"` will be sufficient.

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-ci-index-observability-bundle

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-ci-index-observability-bundle

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-ci-index-observability-bundle

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-ci-index-observability-bundle

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-observability-operator-e2e

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-ci-index-observability-bundle

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-images

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-observability-operator-e2e

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-observability-operator-e2e

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-observability-operator-e2e

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-observability-operator-e2e

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-observability-operator-e2e

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-observability-operator-e2e

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-observability-operator-e2e

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-observability-operator-e2e

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-observability-operator-e2e

**Comment by lihongyan1** [comment]
/pj-rehearse periodic-ci-rhobs-observability-operator-main-periodic-e2e-test-harness

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-observability-operator-e2e

**Comment by lihongyan1** [comment]
/pj-rehearse pull-ci-rhobs-observability-operator-main-observability-operator-e2e

File: ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: observability-operator:0.0.25
+    with: pipeline:observability-operator
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
```
Comment (jan--f) [inline ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main.yaml:35]: Should we test a 4.14 image on a 4.13 cluster?

Comment (jan--f) [reply ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main.yaml:35]: oh my mistake...that is the main point here...please ignore.

**Pull request merged**